File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleYoutubeVideoElement.java
package com.gigigo.orchextra.core.domain.entities.article;

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement;
import java.io.Serializable;

public class ArticleYoutubeVideoElement extends ArticleElement implements Serializable {

  private String source;

  public String getSource() {
    return source;
  }

  public void setSource(String source) {
    this.source = source;
  }
}

File: ocmsdk/src/main/java/com/gigigo/orchextra/core/domain/entities/article/ArticleYoutubeVideoElement.kt
package com.gigigo.orchextra.core.domain.entities.article

import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElement
import com.gigigo.orchextra.core.domain.entities.article.base.ArticleElementRender

class ArticleYoutubeVideoElement : ArticleElement<ArticleYoutubeVideoElementRender>()

class ArticleYoutubeVideoElementRender : ArticleElementRender() {

  var source: String? = null
}
